migrate.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `citizenid` varchar(50) DEFAULT NULL,
+    `plate` varchar(15) NOT NULL,
+    `balance` int(11) NOT NULL DEFAULT 0,
+    `paymentamount` int(11) NOT NULL DEFAULT 0,
+    `paymentsleft` int(11) NOT NULL DEFAULT 0,
+    `financetime` int(11) NOT NULL DEFAULT 0,
+    PRIMARY KEY (`id`),
+    FOREIGN KEY `plate` (`plate`) REFERENCES `player_vehicles` (`plate`) ON DELETE CASCADE ON UPDATE CASCADE,
+    FOREIGN KEY `citizenid` (`citizenid`) REFERENCES `players` (citizenid) ON DELETE CASCADE ON UPDATE CASCADE

```

Comment:
If we're going to keep this, shouldn't it reference the citizenid on the vehicles table instead?

Reply:
The players_vehicles shouldn't even have a citizenid column tbh,
But if i really need to defend this, the citizenid of financing isn't really a reference to the player that owns the vehicle,
it's a reference to the player (character but whatever) that pays for the 'loan'.

Reply:
I'm confused why you say the player_vehicles shouldn't have citizenid? Doesn't a vehicle need to be associated to a player?

Reply:
Don't worry about it, i think vehicles should be separated from players but that's for another time.